CMakeLists.txt

```diff
@@ -364,7 +364,6 @@ set(LIT_ARGS_DEFAULT "-sv")
 if (MSVC OR XCODE)
   set(LIT_ARGS_DEFAULT "${LIT_ARGS_DEFAULT} --no-progress-bar")
 endif()
-set(LLVM_LIT_ARGS "${LIT_ARGS_DEFAULT}" CACHE STRING "Default options for lit")
```

Comment:
I assume this removal wasn't intentional.

Reply:
Good catch, thank you!